roles/client-orcid-idp/files/orcid2urn.php

```diff
@@ -0,0 +1,10 @@
+<?php
+$attributemap = array(
+    // Attributes returned by ORCID
+    'orcid.path'         => 'urn:mace:dir:attribute-def:uid',
+    'orcid.path'         => 'urn:mace:dir:attribute-def:eduPersonPrincipalName',
```

Comment:
Dit is een ongescoped ding, volgens mij.  Kunnen we ePPN niet construeren van <orcid.pad>@<orchid.host> ?

Reply:
Nee, niets werkt out-of-the box. Je zult zelf een client-id moeten registreren vanwege secrets lekken op github

Reply:
ePPN wordt elders gezet